lib/generate-screenshot-file-path.js

```diff
@@ -41,14 +41,18 @@ function defaultScreenshotFilePath(nightwatchClient, basePath, fileName) {
  * @param {String} [fileName] A custom filename for the screenshot
  */
 module.exports = function generateScreenshotFilePath(nightwatchClient, basePath, fileName) {
-    const globalSettings = get(
-        nightwatchClient,
-        'globals.test_settings.visual_regression_settings',
-        null
-    )
-
-    if (globalSettings && globalSettings.generate_screenshot_path) {
-        return globalSettings.generate_screenshot_path(
+    const visualRegressionSettingsPaths = [
+        'globals.visual_regression_settings',
+        'globals.test_settings.visual_regression_settings'
+    ]
+
+    const settings = visualRegressionSettingsPaths.map((settingPath) =>
+        get(nightwatchClient, settingPath, null)
+    ).find((vrtSettings) => vrtSettings)
```

Comment:
I'd change this `find` to `.filter(Boolean)[0]` to make it a little more clear what the logic is intended to do.

Reply:
Thanks!